### PR TITLE
ci: separate the config dump from the configuration command

### PR DESCRIPTION
**Pull request by paulidale**
This avoids using the shell's `&&` and shortens the lines a bit.
It also makes for less scrolling when looking for the config command line.

- [ ] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

File: .github/workflows/coveralls.yml [hunk 1]
```diff
@@ -14,7 +14,10 @@ jobs:
       run: |
         sudo apt-get -yq install lcov
     - name: config
-      run: CC=gcc ./config --banner=Configured --debug --coverage no-asm enable-fips enable-rc5 enable-md2 enable-ssl3 enable-nextprotoneg enable-ssl3-method enable-weak-ssl-ciphers enable-zlib enable-ec_nistp_64_gcc_128 no-shared enable-buildtest-c++ enable-external-tests -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION && perl configdata.pm --dump
+      run: CC=gcc ./config --banner=Configured --debug --coverage no-asm enable-fips enable-rc5 enable-md2 enable-ssl3 enable-nextprotoneg enable-ssl3-method enable-weak-ssl-ciphers enable-zlib enable-ec_nistp_64_gcc_128 no-shared enable-buildtest-c++ enable-external-tests -DPEDANTIC -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
+    - name: config dump
+      run: ./configdata.pm --dump
+
```
Comment (levitte) [inline .github/workflows/coveralls.yml:20]: Is there a reason for this sudden empty line? (not only here, you added it consistently)

Comment (paulidale) [reply .github/workflows/coveralls.yml:20]: Readability.  It's a nice visual separation from the configuration and the "work".
I like paragraph breaks, they are easy on the eyes.

Comment (levitte) [reply .github/workflows/coveralls.yml:20]: To my eyes, this is similar to the line breaks we have between separate jobs, that made it easy to see what constitutes one job at a glance.  With these added line breaks, that's no longer true, you suddenly have to stop and look if the line break actually shows where a new job starts or if it's inside the job description.

On the other hand, I noticed that you added an empty line before the configuration steps in *some* jobs but not others...  so if you insist on this separation, please be consistent with the empty line before the configuration steps.

Comment (paulidale) [reply .github/workflows/coveralls.yml:20]: Most of our CIs only have one job in them.

Comment (paulidale) [reply .github/workflows/coveralls.yml:20]: And windows.yml does it differently.

**Comment by paulidale** [comment]
Ping?

File: .github/workflows/coveralls.yml (hunk 1 above)
Comment (t8m) [reply .github/workflows/coveralls.yml:20]: I'd agree here with @levitte that the extra empty lines you add with this change to the various CI configs should not be added. It does not seem to improve readability to me.

**Comment by paulidale** [comment]
Extra blank lines removed.

**Comment by paulidale** [comment]
The build with CC=gcc is fixing a bug, so there is more than just refactoring involved here.
The default CC is clang on the VMs we're building on.

**Review by t8m — APPROVED** [review]
Given this is refactoring just of the CI job setup and not the actual openssl library code, this IMO does not require any extra formal approval from OTC.

**Comment by paulidale** [comment]
Merged, thanks for the feedback.

**Pull request closed without merging**